### ISSUE TITLE
Add `check-upstream-dashboard.sh` to compatibility check

### ISSUE THREAD

**Issue by amisevsk** [redhat-developer/rh-che#1643]
**Description:**
We currently have a script `./dev-scripts/check-upstream-dashboard.sh` which we use on upstream releases to check if rh-che dashboard updates are necessary. It would be nice to have this included in the compatibility check if possible.

Some caveats:
- The script works between maven releases, so it would be a little tricky to use when checking SNAPSHOT -> SNAPSHOT changes (maybe it could be used to issue a warning that updates will need to be done? We currently patch the dashboard files only once the upstream version is released, i.e. not a snapshot).
- The script would need to be updated to return a status on failure, it currently only prints changed files.

**Comment by amisevsk** [comment]
Given the limitations of the script, it might not make perfect sense for the compatibility check, in which case I'm okay with not including it.

**Comment by ibuziuk** [comment]
Closing as outdated. We no longer do the upstream UD patching
